### PR TITLE
Update Relese.md

### PR DESCRIPTION
**Pull request by venkat2469**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -89,6 +67,40 @@
...
+
+* Moved all saving-related utilities to a new namespace, `keras.saving`, for example: `keras.saving.load_model`, `keras.saving.save_model`, `keras.saving.custom_object_scope`, `keras.saving.get_custom_objects`, `keras.saving.register_keras_serializable`,`keras.saving.get_registered_name` and `keras.saving.get_registered_object`. The previous API locations (in `keras.utils` and `keras.models`) will be available indefinitely, but we recommend you update your code to point to the new API locations.
+ * Improvements and fixes in Keras loss masking:
+    * Whether you represent a ragged tensor as a `tf.RaggedTensor` or using [keras masking](https://www.tensorflow.org/guide/keras/masking_and_padding), the returned loss values should be the identical to each other. In previous versions Keras may have silently ignored the mask.
+ * If you use masked losses with Keras the loss values may be different in TensorFlow `2.12` compared to previous versions.
+ * In cases where the mask was previously ignored, you will now get an error if you pass a mask with an incompatible shape.
+
+### Major Features and Improvements     
+
+ tf.keras:
```
Comment (huberalberti) [inline RELEASE.md:84]: Add quotes `tf.keras`:

File: RELEASE.md [hunk 2]
```diff
@@ -89,6 +67,40 @@
 *  `tf.config.experimental.mlir_bridge_rollout`
     *   Removed enums `MLIR_BRIDGE_ROLLOUT_SAFE_MODE_ENABLED` and `MLIR_BRIDGE_ROLLOUT_SAFE_MODE_FALLBACK_ENABLED` which are no longer used by the tf2xla bridge
 
+## Keras
+
+ Keras is a framework built on top of the TensorFlow framework. See more details on the Keras [website](https://keras.io/).
+
+### Breaking Changes
+
```
Comment (huberalberti) [inline RELEASE.md:75]: Add `tf.keras`: at the beginning and indent the rest under it

**Review by huberalberti — CHANGES_REQUESTED** [review]

File: RELEASE.md [hunk 3]
```diff
@@ -18,40 +20,17 @@
...
+*  Support for Python 3.11 has been added.
+*  Support for Python 3.7 has been removed. We are not releasing any more patches for Python 3.7.
 
 *   `tf.lite`:
 
     *   Add 16-bit float type support for built-in op `fill`.
     *   Transpose now supports 6D tensors.
     *   Float LSTM now supports diagonal recurrent tensors: https://arxiv.org/abs/1903.08023
 
-*   `tf.keras`:
```
Comment (rishikasinha-tf) [inline RELEASE.md:39]: Let's not bullet "tf.keras" line

File: RELEASE.md [hunk 4]
```diff
@@ -89,6 +68,43 @@
...
     *   Removed enums `MLIR_BRIDGE_ROLLOUT_SAFE_MODE_ENABLED` and `MLIR_BRIDGE_ROLLOUT_SAFE_MODE_FALLBACK_ENABLED` which are no longer used by the tf2xla bridge
 
+## Keras
+
+ Keras is a framework built on top of the TensorFlow framework. See more details on the Keras [website](https://keras.io/).
+
+### Breaking Changes
+
+
+*   `tf.keras`:
```
Comment (rishikasinha-tf) [inline RELEASE.md:78]: Let's not bullet "tf.keras" line

File: RELEASE.md [hunk 5]
```diff
@@ -89,6 +68,43 @@
...
+
+* Moved all saving-related utilities to a new namespace, `keras.saving`, for example: `keras.saving.load_model`, `keras.saving.save_model`, `keras.saving.custom_object_scope`, `keras.saving.get_custom_objects`, `keras.saving.register_keras_serializable`,`keras.saving.get_registered_name` and `keras.saving.get_registered_object`. The previous API locations (in `keras.utils` and `keras.models`) will be available indefinitely, but we recommend you update your code to point to the new API locations.
+ * Improvements and fixes in Keras loss masking:
+    * Whether you represent a ragged tensor as a `tf.RaggedTensor` or using [keras masking](https://www.tensorflow.org/guide/keras/masking_and_padding), the returned loss values should be the identical to each other. In previous versions Keras may have silently ignored the mask.
+ * If you use masked losses with Keras the loss values may be different in TensorFlow `2.12` compared to previous versions.
+ * In cases where the mask was previously ignored, you will now get an error if you pass a mask with an incompatible shape.
+
+### Major Features and Improvements     
+
+*   `tf.keras`:
```
Comment (rishikasinha-tf) [inline RELEASE.md:88]: Let's not bullet "tf.keras" line

**Review by huberalberti — APPROVED** [review]
LGTM

**Review by rishikasinha-tf — APPROVED** [review]

**Pull request merged**